### PR TITLE
Fix Chain Sync Issues

### PR DESCRIPTION
**Pull request by ejfitzgerald**

### CODE REVIEW

File: libs/ledger/include/ledger/chain/constants.hpp [hunk 1]
```diff
@@ -22,6 +22,8 @@
 namespace fetch {
 namespace ledger {
 
+static const uint64_t FINALITY_PERIOD = 50;
```
Comment (pbukva) [inline libs/ledger/include/ledger/chain/constants.hpp]: You can replace `static const` with `constexpr`:

```cpp
constexpr uint64_t FINALITY_PERIOD = 50;
```

File: apps/constellation/constellation.cpp [hunk 2]
```diff
@@ -336,9 +336,12 @@ void Constellation::Run(UriList const &initial_peers)
   // monitor loop
   while (active_)
   {
+    // determine the status of the main chain server
+    bool const is_in_sync =
+        ledger::MainChainRpcService::State::SYNCHRONISED == main_chain_service_->state();
```
Comment (n-hutton) [inline apps/constellation/constellation.cpp:341]: Minor: this is a bit verbose

**Review by n-hutton — DISMISSED** [review]

**Review by pbukva — CHANGES_REQUESTED** [review]
LGTM
Just look for inline comments and let me know (nothing critical only tweaks, PR can go as is).

**Review by pbukva — DISMISSED** [review]
LGTM

**Review by pbukva — APPROVED** [review]

**Pull request merged**